graph-commons/src/test/scala/ch/datascience/rdfstore/triples/entities/Agent.scala

```diff
@@ -28,10 +28,10 @@ object Agent {
   {
     "@id": $id,
     "@type": [
-      "prov:SoftwareAgent",
+      "http://www.w3.org/ns/prov#SoftwareAgent",
```

Comment:
why expand `prov` everywhere? It looks like it's in the query header anyway?

Reply:
These entities are only used to generate triples for testing purposes. I needed to make them as close as possible to what `renku log` produces as triples generation does some tweaks to the JSON it receives and so it has to be very accurate.